Presentation.Web/app/components/it-contract/tabs/it-contract-tab-main.view.html

```diff
@@ -58,11 +58,11 @@ <h4>{{contract.name}}</h4>
 
                         <!--Criticality type-->
                         <div class="form-group">
-                            <label for="criticality" class="control-label small">Kritikalitet</label>
+                            <label for="criticality" class="control-label small">Kritikalitet <field-help-text data-key="it-contract.edit.main.criticality"></field-help-text></label>
                             <single-select-select2 id="criticality"
```

Comment:
I wanted to add the help text to the directive (like to the select2Option), but the label is defined in the view so I had to add the help text here. I guess to add the help text to the directive we would also have to move the label there